src/test/java/uk/gov/hmcts/reform/em/orchestrator/automatedbundling/configuration/LocalConfigurationLoaderTest.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.reform.em.orchestrator.automatedbundling.configuration;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class LocalConfigurationLoaderTest {
+    private final LocalConfigurationLoader loader = new LocalConfigurationLoader(new ObjectMapper(new YAMLFactory()));
+
+    @Test
+    public void load() {
+        BundleConfiguration config = loader.load("example.yaml");
+
+        assertEquals(config.title, "New bundle");
+        assertEquals(config.folders.get(0).name, "Folder 1");
+        assertEquals(config.folders.get(1).name, "Folder 2");
+    }
+
+    @Test(expected = BundleConfigurationException.class)
+    public void loadMissingConfig() {
+        loader.load("does-not-exist.yaml");
+    }
+}
```

Comment:
how come checkstyle's not being annoying about no newline at end of file?! It'd 100% get me for it

Reply:
it's currently failing, waiting for your fixes before I look at it